core-plugins/src/main/java/co/cask/hydrator/plugin/transform/ProjectionTransform.java

```diff
@@ -127,10 +134,11 @@ public void transform(StructuredRecord valueIn, Emitter<StructuredRecord> emitte
     StructuredRecord.Builder builder = StructuredRecord.builder(outputSchema);
     for (Schema.Field inputField : inputSchema.getFields()) {
       String inputFieldName = inputField.getName();
-      if (fieldsToDrop.contains(inputFieldName)) {
+      if (!fieldsToKeep.isEmpty() && (!fieldsToKeep.contains(inputFieldName)) {
```

Comment:
There was a jira but it didn't specify clearly. I assume both drop and keep can be empty to not have the user add fields to keep everytime they're not using drop?